index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/png" href="/assets/favicon.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Schulcloud tldraw</title>
```

Comment:
I think that needs to be checked by your BA / PO... the Software is formally called "Schulcloud Verbund Software".

Reply:
You're right, changed it to just 'tldraw' for now and asked BAs about it.